nsdb-cluster/src/main/scala/io/radicalbit/nsdb/cluster/NSDBAkkaCluster.scala

```diff
@@ -33,54 +33,58 @@ import scala.concurrent.ExecutionContextExecutor
 /**
   * Creates the [[ActorSystem]] based on a configuration provided by the concrete implementation
   */
-trait NSDBAkkaCluster { this: NsdbConfig =>
+trait NSDBAkkaCluster { this: NsdbConfig with NSDBAActors =>
```

Comment:
`NSDBAACtors` states for `NSDB Akka Actors`?

Reply:
ah sorry that's a typo. there's one too many A